shotover/src/lib.rs

```diff
@@ -75,6 +76,7 @@ pub mod tcp;
 pub mod tls;
 mod tracing_panic_handler;
 pub mod transforms;
+pub mod unix_socket_server;
```

Comment:
this is specifically for hot reloading, so lets name this something like `hotreload_server`